Protos/V1/results_common.proto

```diff
@@ -104,18 +113,139 @@ message ListResultsResponse {
 *  Request for getting the id of the task that should create this result
 */
 message GetOwnerTaskIdRequest {
-  string session_id = 1;
-  repeated string result_id = 2;
+  string session_id = 1; /** The session ID. */
+  repeated string result_id = 2; /** The list of result ID/name. */
 }
 
 /*
 * Response for getting the id of the task that should create this result
 */
 message GetOwnerTaskIdResponse {
   message MapResultTask {
-    string result_id = 1;
-    string task_id = 2;
+    string result_id = 1; /** The result ID/name. */
+    string task_id = 2; /** The owner task ID associated to the result. */
   }
   repeated MapResultTask result_task = 1;
-  string session_id = 2;
+  string session_id = 2; /** The session ID. */
+}
+
+/*
+* Request for creating results without data
+*/
+message CreateResultsMetaDataRequest {
+  /**
+  * A result to create.
+  */
+  message ResultCreate {
+    string name = 1; /** The result name. Given by the client. */
```

Comment:
Why have a type with only 1 field ? Futureproofing ?

Reply:
Yes, this is common practice that rpc have only custom types